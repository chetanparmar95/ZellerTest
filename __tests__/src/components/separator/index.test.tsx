import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Separator from '../../../../src/components/separator';

it('renders correctly', () => {
  const wrapper = renderer.create(<Separator />);
  expect(wrapper).toMatchSnapshot();
});
