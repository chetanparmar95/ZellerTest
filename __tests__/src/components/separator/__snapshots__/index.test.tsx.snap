// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "rgb(234,242,250)",
      "height": 1,
      "marginVertical": 16,
      "width": "100%",
    }
  }
/>
`;
